.evergreen/install-dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
```

Comment:
Check if this file has been modified from drivers-evergreen-tools. If not, let's use the stock script and remove this.